src/toil/batchSystems/singleMachine.py

```diff
@@ -221,9 +221,16 @@ def daddy(self):
                             # cores/memory/disk individually)?
                             self.inputQueue.put(args)
                             break
+                        elif result is not False:
+                            #Result is a PID
+
+                            if self._outbox is not None:
+                                # Annotate the job with the PID generated.
+                                self._outbox.publish(
+                                   BatchSystemMessage(jobID, result, self.__class__.__name__))
```

Comment:
At this point `result` is a numeric PID, right? But `BatchSystemMessage` needs to hold a string. So don't we need a `str()` call to convert?